core/src/main/antlr/xtdb/antlr/SqlLexer.g4

```diff
@@ -265,9 +267,12 @@ PASSWORD: 'PASSWORD' ;
 PATCH : 'PATCH' ;
 PERCENT_RANK : 'PERCENT_RANK' ;
 PERIOD : 'PERIOD' ;
+PG_CATALOG_DEFAULT : 'pg_catalog.default' ;
```

Comment:
I doubt this is a token in the language, unless we're going to specify every collation as a token - I suspect collation wants to be `(schemaName '.')? identifier`?